### PR TITLE
Merge from dev-https to master.

### PR DESCRIPTION
**Pull request by mc-hamster**
@geeksville I keep breaking the CI. This time it says that it failed the tbeam build because it couldn't find the esp32_https_server library, even though it is in platform.ini and in the right place (you put it there). The file is the same on my local environment and on the dev-https branch.

### CODE REVIEW

**Comment by geeksville** [comment]
hmm strange - I'll pull today and take a look.  sorry about that (btw - the wifi at my hotel might go out again and I can't switch sims on my phone to my google fi because I temporarily need to keep using the 'tracked' sim from taiwan until my quarantine is over (sunday night(

File: src/meshwifi/meshhttp.h [hunk 1]
```diff
@@ -15,8 +17,133 @@ void notifyWebUI();
...
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+
+// Length of the binary data
+const int FAVICON_LENGTH = 2238;
+
+class httpAPI : public PhoneAPI
```
Comment (geeksville) [inline src/meshwifi/meshhttp.h:138]: I think this is fine to merge, but class names normally start with an upper case letter.  So good to change that in a follow up commit. (i.e. HttpAPI)

**Review by geeksville — APPROVED** [review]
looks good to me (though see my comment about HttpAPI

File: src/meshwifi/meshhttp.h (hunk 1 above)
Comment (mc-hamster) [reply src/meshwifi/meshhttp.h:138]: I'll change the case for the class. ASAP.

**Pull request merged**